tests/integration/test_dbapi_integration.py

```diff
@@ -294,7 +294,7 @@ def test_datetime_with_utc_time_zone_query_param(trino_connection):
     rows = cur.fetchall()
 
     assert rows[0][0] == params
-    assert cur.description[0][1] == "timestamp with time zone"
+    assert cur.description[0][1] == "timestamp(6) with time zone"
```

Comment:
Note that only milliseconds are currently supported. If you would change the test case to more precise values over ms, eg. `params = datetime(2020, 1, 1, 16, 43, 22, 320258)`, you would see that 258 part is lost, because of #42.

IMHO providing a value with a precision that is not actually supported right now seems not so useful.

Reply:
Actually the whole 6 digits are preserved now.